packages/builder/src/pages/builder/apps/index.svelte

```diff
@@ -34,12 +34,16 @@
   const publishedAppsOnly = app => app.status === AppStatus.DEPLOYED
 
   $: publishedApps = $apps.filter(publishedAppsOnly)
-
+  $: isCloud = $admin.cloud
   $: userApps = $auth.user?.builder?.global
     ? publishedApps
     : publishedApps.filter(app =>
         Object.keys($auth.user?.roles).includes(app.prodId)
       )
+
+  function getUrl(app) {
+    return !isCloud ? `/app/${encodeURIComponent(app.name)}` : `/${app.prodId}`
```

Comment:
How come we need `/app` prefix for the short name but not the id?
Would you be able to paste an example link for the short and long form?

Reply:
the `/app` is to make it easier to direct to the server instance through envoy, the same way `/db` or `/global` go to CouchDB/worker respectively.

We had a special case for the app ID in Envoy, directing anything from `/app_` to the server instance, so that it could work out how to serve the client etc, but if we are using the app name we need some consistent way to direct to the server instance (the default is MinIO because it didn't play nice with a path).

examples would be:
long: `localhost:10000/app_9d739324a920463facbf13bd3a4560b7`
short: `localhost:10000/app/test`

The only way we could get rid of the `/app` is if we made it so that the server was the default and somehow got MinIO to behave under these conditions (probably would need to proxy everything through the server) and then we'd need to make sure that is backwards compatible. Personally I think its probably safer to have everything going to a server under a path like `/app` as it gives us a lot of consistent control. The `/app_` special case does essentially the same thing, its just that we know for certain all app IDs will start with it.

Reply:
Got it, thanks! I actually prefer the /app prefix, it’d be nice to just drop the special logic for /app_ in envoy and have the routing be consistent, keeping both approaches is fine too and probably required for backwards compatibility

Reply:
Have to agree I'd like to drop it too, the only reason it was there was for backwards compatibility, it was built that way during the transition from the original electron builder to what it is today as the OG server handled routing that way. I think we leave the rule in place for a little while until the short hand is fully adopted and then we could get rid of it.